src/api/java/Context.java

```diff
@@ -3878,7 +3870,7 @@ public void updateParamValue(String id, String value)
     }
 
     protected long m_ctx = 0;
-    protected static Object creation_lock = new Object();
+    protected static final Object creation_lock = new Object();
```

Comment:
It's a compile-time annotation.
It will not allow to compile a code which changes the creation lock.